prototyps/Eclipse GUI Prototype/src/de/uka/ipd/sdq/beagle/Activator.java

```diff
@@ -0,0 +1,50 @@
+package de.uka.ipd.sdq.beagle;
+
+import org.eclipse.ui.plugin.AbstractUIPlugin;
+import org.osgi.framework.BundleContext;
+
+/**
+ * The activator class controls the plug-in life cycle
+ */
+public class Activator extends AbstractUIPlugin {
+
+	// The plug-in ID
+	public static final String PLUGIN_ID = "Eclipse_GUI_Protoype"; //$NON-NLS-1$
```

Comment:
[This](http://stackoverflow.com/questions/654037/what-does-non-nls-1-mean).

I'll remove it, this comment doesn't make sense in our context. (I used anoteher Clean-Up configuration, as the one for our project was not finished, when I wrote this.)